content/spire/getting-started-linux.md

```diff
@@ -91,25 +91,12 @@ To install the server and agent:
 2. Add `spire-server` and `spire-agent` to your $PATH for convenience:
 
 	```shell
-	ln -s /opt/spire/spire-server /usr/bin/spire-server
-	ln -s /opt/spire/spire-agent /usr/bin/spire-agent
+	sudo ln -s /opt/spire/spire-server /usr/bin/spire-server
+	sudo ln -s /opt/spire/spire-agent /usr/bin/spire-agent
 	```
 
 ## Step 4: Configure the Server {#step-4}
 
-To configure the server you:
-
-1. Configure the trust domain
-2. Configure the server certificate authority (CA), which might include configuring an UpstreamCA plugin 
-3. Configure the node attestation plugin
-4. Configure a default **.data** directory for persisting data
```

Comment:
I think we should keep this part of the text...? Some of these steps (e.g. 1 and 3) are still valid, and it looks like we added one more (configure data directory)?

In general, we should probably try to avoid having headers/sections with no text in them

Reply:
Fair enough. I removed it since the agent did not have a similar header. I will restore this and add one to the agent section.